mlos_bench/mlos_bench/service/config_persistence.py

```diff
@@ -140,7 +140,12 @@ def prepare_class_load(self, config: Dict[str, Any],
             class_config[key] = global_config[key]
 
         for key in class_params.intersection(config.get("resolve_path", [])):
-            class_config[key] = self.resolve_path(class_config[key])
+            if isinstance(class_config[key], str):
+                class_config[key] = self.resolve_path(class_config[key])
+            elif isinstance(class_config[key], (list, tuple)):
```

Comment:
```suggestion
            elif isinstance(class_config[key], Iterable):
```

I think this should work and is more generic.

Reply:
But fine as is too.

Reply:
nah, that will accept a `dict` but work incorrectly. I'd rather be more strict